desktop-widgets/btdeviceselectiondialog.ui

```diff
@@ -207,16 +154,99 @@
      </layout>
     </widget>
    </item>
-   <item row="3" column="0">
-    <widget class="QLabel" name="dialogStatus">
-     <property name="text">
-      <string/>
+   <item row="0" column="0">
+    <widget class="QLabel" name="discoveredDevicesLabel">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
      </property>
-     <property name="wordWrap">
-      <bool>true</bool>
+     <property name="font">
+      <font>
+       <weight>75</weight>
+       <bold>true</bold>
+      </font>
+     </property>
+     <property name="text">
+      <string>Discovered devices</string>
      </property>
     </widget>
    </item>
+   <item row="1" column="0" rowspan="2">
+    <layout class="QVBoxLayout" name="remoteDevicesSection">
+     <item>
+      <widget class="QListWidget" name="discoveredDevicesList">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <layout class="QHBoxLayout" name="bleModeSection">
+       <item>
+        <widget class="QLabel" name="bleModeLabel">
+         <property name="text">
+          <string>Bluetooth low energy (BLE) mode</string>
```

Comment:
~~can we rename this to be:~~
~~`Bluetotoh Low Energy (BLE) mode`~~

~~my reasoning for the capitalization - that's how it's spelled as a technology name.~~

Reply:
actually, bellow we have a set of `Bluetooth` choices, which also include `classical`.
the label above the combo box should be just:
`Bluetooth mode`